aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -30,7 +30,10 @@ module aptos_token::token {
 
     const MAX_COLLECTION_NAME_LENGTH: u64 = 128;
     const MAX_NFT_NAME_LENGTH: u64 = 128;
-    const MAX_URI_LENGTH: u64 = 512;
+    const MAX_COLLECTION_URI_LENGTH: u64 = 512;
+
+    // to support SVG as meta data, the token uri limits should be increased.
+    const MAX_TOKEN_URI_LENGTH: u64 = 51200;
```

Comment:
Store the data in off-chain storage like arweave or stored in the default properties in tokendata?

Reply:
onchain with RFC2986: https://www.rfc-editor.org/rfc/rfc3986 support data URIs.

Reply:
it's part of ERC721 standard. 

`    /// @notice A distinct Uniform Resource Identifier (URI) for a given asset.
    /// @dev Throws if `_tokenId` is not a valid NFT. URIs are defined in RFC
    ///  3986. The URI may point to a JSON file that conforms to the "ERC721
    ///  Metadata JSON Schema".
    function tokenURI(uint256 _tokenId) external view returns (string);`

https://eips.ethereum.org/EIPS/eip-721